test.js

```diff
@@ -331,4 +334,21 @@ describe('javascript-stringify', function () {
       test(obj, '{a:{b:{}}}', null, { maxDepth: 2, references: true })
     );
   });
+  describe('property based', function () {
+    var customEval = function (repr) {
```

Comment:
Would it make sense to just use `eval` for now? I'm not sure we need the additional wrapper.

Reply:
I think I had an issue with eval, but I will try again ;)

Reply:
Calling `console.log(eval("{}"))` prints `undefined`.
Calling ` console.log(eval("(function () { return {}; })()"));` does work.

So I can either have:

```js
var customEval = function (repr) {
  return eval("(function () { return " + repr + "; })()");
};
```

or

```js
var customEval = function (repr) {
  return Function("return " + repr)();
};
```

Reply:
That makes sense, since it's a block. All good 👍